pkg/profiler/cpu/maps.go

```diff
@@ -277,7 +282,14 @@ func (m *bpfMaps) clean() error {
 
 // setUnwindTable updates the unwind tables with the given unwind table.
 func (m *bpfMaps) setUnwindTable(pid int, ut unwind.UnwindTable) error {
-	buf := new(bytes.Buffer)
+	buf := m.pool.Get().(*bytes.Buffer)
+	keyBuf := m.pool.Get().(*bytes.Buffer)
+	defer func() {
+		buf.Reset()
```

Comment:
I prefer to do resets when getting as then I can be sure they're reset and we don't have to handle various edge cases.